components/partials/sign_in/ViaQuestions.tsx

```diff
@@ -0,0 +1,25 @@
+import { ChildrenComponent as CC } from "components/brickroom/utils";
+import { useTranslation } from "next-i18next";
+
+//
+
+export namespace ViaQuestionsNS {
+  export interface Props {}
+}
+
+//
+
+export default function ViaQuestions(props: CC<ViaQuestionsNS.Props>) {
+  const { t } = useTranslation("signInProps", { keyPrefix: "viaQuestions" });
+
+  return (
+    <div>
+      {/* Intro */}
+      <h2>{t("title")}</h2>
+      <p className="mt-2 mb-6">{t("description")}</p>
+
+      {/* Form */}
```

Comment:
If you want this to be always a Form, you want to enforce it with typings

Reply:
This component is used only as a partial in `sign_in`, and its sole purpose is to wrap the `Questions` component (that is used elsewhere without that title and description).

If I put a comment like `{/* Here goes the Questions component */}` can do?